app/src/main/java/com/example/keniphhomemaintenance/ui/cookbook/KeniphKeyboardActions.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.keniphhomemaintenance.ui.cookbook
+
+enum class KeniphKeyboardActions {
+    NONE,
```

Comment:
You're getting ahead of yourself again - don't add options/functions/anything that you're not going to implement immediately. What's the point of adding unused code? It's just confusing to other people who may need to deal with your code.